.github/workflows/integration-tests.yml

```diff
@@ -10,12 +10,15 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
After the integration test refactoring, as a side effect, the CI runs are a bit better organized in the UI:
<img width="684" height="1946" alt="image" src="https://github.com/user-attachments/assets/625691fe-18f2-4307-825e-3e78bf2fd0bc" />

Reply:
Pipeline: 
<img width="2194" height="1086" alt="image" src="https://github.com/user-attachments/assets/25961bad-af21-4aa3-96d1-29bf55542939" />